network/src/main/scala/no/ndla/network/clients/FeideApiClient.scala

```diff
@@ -95,6 +104,21 @@ trait FeideApiClient {
       })
     }
 
+    private def findCounty(feideGroups: Seq[FeideGroup]): Try[String] = {
+      feideGroups.find(group => group.membership.primaryAffiliation.isDefined) match {
```

Comment:
Klarer ikke se noe sted i doc'en at `primaryAffiliation` er det som bestemmer dette. Burde vi spørre feide om hvilken gruppe vi burde velge? @gunnarvelle 

Og kanskje evt ha en fallback til den første (e.l) dersom vi ikke klarer å finne en preferert en?
```suggestion
      val primaryGroup = feideGroups.find(group => group.membership.primaryAffiliation.isDefined)
      val anyGroup = primaryGroup.orElse(feideGroups.headOption)
      anyGroup match {
```